src/netstandard/ref/System.Collections.ObjectModel.cs

```diff
@@ -26,9 +27,15 @@ public partial class Collection<T> : System.Collections.Generic.ICollection<T>,
         public int IndexOf(T item) { throw null; }
         public void Insert(int index, T item) { }
         protected virtual void InsertItem(int index, T item) { }
+        protected virtual void InsertItemsRange(int index, System.Collections.Generic.IEnumerable<T> collection) { }
+        public void InsertRange(int index, System.Collections.Generic.IEnumerable<T> collection) { }
         public bool Remove(T item) { throw null; }
         public void RemoveAt(int index) { }
         protected virtual void RemoveItem(int index) { }
+        protected virtual void RemoveItemsRange(int index, int count) { }
```

Comment:
Any reason why this and APIs bellow don't use Range type?

Reply:
Because this API was [approved](https://github.com/dotnet/corefx/issues/10752) 3 years ago but only recently implemented. Not sure it's worth going back to changing the design. We can always add new overloads.